content/ADR-204-comms-architecture.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: adr
+adr: 204
+title: Comms Architecture for new client
+date: 2024-08-21
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+authors:
+  - pentreathm
+  
+---
+
+## Abstract
+
+The current Decentraland client communication protocol effectively supports user scaling but has inherent design limitations that restrict certain functionalities in Genesis City, such as content casting for in-world streaming, serverless multiplayer experiences, and being able to handle scene-hosted events with speaker moderators. This ADR explores the architectural decisions behind a new communication protocol designed to overcome these limitations. 
+
+## Context, Reach & Prioritization 
+
+The new architecture, incompatible with the existing implementation, will be deployed in a forthcoming client version and become the new standard.The objective is to incorporate the following functionalities without overburdening the client implementation with additional complexities, while maintaining the ability to stream the world and visualize users in surrounding areas. This entails:
```

Comment:
What do you think of linking the ADR with `existing implemenation`?

Reply:
done